packages/mobile/src/firebase/notifications.ts

```diff
@@ -47,6 +48,7 @@ function* handlePaymentRequested(
     firebasePendingRequestUid: paymentRequest.uid,
     recipient: targetRecipient,
     amount: new BigNumber(paymentRequest.amount),
+    currency: Currency.Dollar,
```

Comment:
Should we use `paymentRequest.currency` instead? or we decided it's always in dollars for now?

Reply:
So, for requests we show the local currency, so it doesn't really matter which currency we save in Firebase. 

The way this works is:
- requester requests X MXN (local currency) from requestee. We save in Firebase using cUSD in the current rate.
- requestee sees in their own local currency (BRL for example) the cUSD amount to send. When trying to send in the confirmation screen they can switch to cEUR instead and we calculate how much with the cEUR/cUSD exchange rate.

This will be done in a future not yet opened PR that addresses Payment requests. This has a potential issue that I reported [on Slack](https://celo-org.slack.com/archives/C018VLA3YAK/p1623775292095000)

[Here's the Figma mockup](https://www.figma.com/file/oLgp5ZQamOR1vHRWHMcOpv/Celo-Euro?node-id=1%3A3)